docs/building.md

```diff
@@ -120,9 +124,9 @@ export PATH=/path/to/mlir-air/install-pcie/bin:${PATH}
 
 The instructions for building the hardware platform designs are found in the mlir-air/platforms directory:
 
-- [xilinx_vck190_air](platforms/xilinx_vck190_air)
-- [xilinx_vck5000_air](platforms/xilinx_vck5000_air)
-- [xilinx_vck5000_air_scale_out](platforms/xilinx_vck5000_air_scale_out)
+- [xilinx_vck190_air](../platforms/xilinx_vck190_air)
```

Comment:
This is somewhat problematic and keeps arising. @fifield has explained this to me in the past - the links work correctly on github.io (https://xilinx.github.io/mlir-air/building.html) but don't work correctly when browsing the repo directly. No known solution at this time.

Reply:
Do the relatively paths not work on github.io?

Reply:
The relative paths do not work and the preference has been to have it work in the generated docs instead of browsing the repo. There is some directory reorganization needed to fix this. One option is to keep the platform docs in the docs area instead of in the respective platform folders. The current [deployment code](https://github.com/Xilinx/mlir-air/blob/main/.github/workflows/generateDocs.yml#L97).

Reply:
Thanks for pointing that out didn't realize that. The organizational structure sounds like a separate PR, I will change the docs back to what they were.

Reply:
Addressed in [447d44f](https://github.com/Xilinx/mlir-air/pull/250/commits/447d44ff94c68846fea0cd8344ac75836bb9946f).